atlp-rwanda/ecommerce-app-cogito-fe
# #FN-32 feature: Implement Notification Pane

# Trello Board
- https://trello.com/c/ZmpQYe6f
# PR Description
This PR contains the implementation of In-App Notification and the notification pane  feature.
# Description of tasks that were expected to be completed
- [x] All authenticated and authorized user should be able to access the in-app notifications
- [x] All users should see the notifications pane when they interact with the notifications icon.
- [x] The Notification pane should be well designed and user friendly to use
- [x] Allow users to take necessary actions like mark notification as read and deleting them

# Description of tasks that were expected to be completed and that are not yet completed
# How this may be tested?
To Test this PR
0. clone the project on `https://github.com/atlp-rwanda/ecommerce-app-cogito-fe.git`
2. run `yarn install` to install all required dependencies
3. run `yarn run dev`
5. Go to the login page for authentication or create a new account if you don't have one
6. After successful authentication in the navbar you can see a notification bell and you will have all your notification displayed there.
# Screenshots (If appropriate/applicable)
- n/a
# Please check this Checklist before you submit your PR:
